src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/DrawJudgeOrderCallbackHandlerTest.java

```diff
@@ -91,455 +103,689 @@
+
+                    assertThat(response.getData())
+                        .contains(entry("eyewitnessUploadForParty", eyewitnessUploadForParty));
+                }
+
+                @Test
+                void shouldUseDefaultPaperDeterminationWhenNewCcdCase() {
+                    AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse)
+                        drawJudgeOrderCallbackHandler.handle(callbackParams);
+
+                    assertThat(response.getData()).contains(entry("paperDetermination", CCDYesNoOption.NO));
+                }
+
+                @Test
+                void shouldUseSavedPaperDeterminationWhenExistingCcdCase() {
+                    CCDYesNoOption paperDetermination = YES;
+
+                    ccdCase.setPaperDetermination(paperDetermination);
+
+                    AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse)
+                        drawJudgeOrderCallbackHandler.handle(callbackParams);
+
+                    assertThat(response.getData()).contains(entry("paperDetermination", paperDetermination));
+                }
+
+                @Test
+                void shouldUseDefaultEstimatedHearingDurationWhenNewCcdCase() {
+                    AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse)
+                        drawJudgeOrderCallbackHandler.handle(callbackParams);
+
+                    assertThat(response.getData()).contains(entry("estimatedHearingDuration", null));
+                }
+
+                @Test
+                void shouldUseSavedEstimatedHearingDurationWhenExistingCcdCase() {
+                    CCDHearingDurationType estimatedHearingDuration = CCDHearingDurationType.TWO_HOURS;
+
+                    ccdCase.setEstimatedHearingDuration(estimatedHearingDuration);
+
+                    AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse)
+                        drawJudgeOrderCallbackHandler.handle(callbackParams);
+
+                    assertThat(response.getData())
+                        .contains(entry("estimatedHearingDuration", estimatedHearingDuration));
+                }
+
+                @Test
+                void shouldUseDefaultOtherDirectionsWhenNewCcdCase() {
+                    AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse)
+                        drawJudgeOrderCallbackHandler.handle(callbackParams);
+
+                    assertThat(response.getData()).contains(entry("otherDirections", Collections.emptyList()));
+                }
+
+                @Test
+                void shouldUseSavedOtherDirectionsWhenExistingCcdCase() {
+                    List<CCDCollectionElement<CCDOrderDirection>> otherDirections = ImmutableList.of(
+                        CCDOrderDirection.builder()
+                            .sendBy(LocalDate.parse("2020-10-11"))
```

Comment:
QA:Note Why this date? and hardcoded? - maybe something in the future to calculate

Reply:
This is a test. The hardcoded date is test data.